core-war/src/main/webapp/util/javaScript/vuejs/components/comments/silverpeas-comments.js

```diff
@@ -0,0 +1,257 @@
+          componentId: {
+            'type': String,
+            'mandatory': true
+          },
+          resourceId: {
+            'type': String,
+            'mandatory': true
+          },
+          resourceType: {
+            'type': String,
+            'mandatory': true
+          },
+          indexed: {
+            'type': Boolean,
+            'default': true
+          },
+          user: {
+            'type': Object,
+            'mandatory': true
+          }
+        },
+        data : function() {
+          return {
+            comments: [],
+            url: undefined,
+            updatePopin: undefined,
+            commentText: '',
+            updatedCommentText: ''
+          };
+        },
+        created : function() {
+          sp.ajaxRequest(webContext +
+              '/services/bundles/settings/org/silverpeas/util/comment/Comment')
+              .send()
+
+          this.url = webContext + '/services/comments/' + this.componentId + '/' +
+              this.resourceType + '/' + this.resourceId;
+          this.loadComments();
+        },
+        methods : {
+          /**
+           * Validates the specified text satisfies the requirement to be used as a comment's
+           * content.
+           * @param text {String} the text to validate.
+           * @returns {boolean} true if the text is valid, false otherwise.
+           */
+          validateText : function(text) {
+            if (text === null || text === undefined || $.trim(text).length === 0) {
+              SilverpeasError.add(this.messages.commentErrorSingleCharAtLeast);
+            } else if (!isValidTextArea(text)) {
+              SilverpeasError.add(this.messages.commentErrorFieldTooLong);
+            }
+            return !SilverpeasError.show();
+          },
+          /**
+           * Loads all the comments on the given Silverpeas resource from the server.
+           * @returns {Promise} the promise of an HTTP response.
+           */
+          loadComments: function() {
+            return sp.ajaxRequest(this.url)
+                .sendAndPromiseJsonResponse()
+                .then(function(comments) {
+                  this.comments = comments;
+                  this.emitChange('listing', comments);
+                }.bind(this));
+          },
+          /**
+           * Creates a new comment on the given Silverpeas resource for the current user from the
+           * text written in the corresponding text area.
+           */
+          createComment: function() {
+            if (this.validateText(this.commentText)) {
+              const comment = {
+                author : {
+                  id : this.user.id
+                },
+                componentId : this.componentId,
+                resourceId : this.resourceId,
+                resourceType : this.resourceType,
+                indexed : this.indexed,
+                text : this.commentText,
+                textForHtml : this.commentText
```

Comment:
`textForHtml` seems not used.

Reply:
In this new version of the comments widget, it seems not required. Before removing it, I have to test when the text of a comment contains HTML or javascript code; just to check nothing is wrong